src/fractal_faim_hcs/MetaSeriesUtils_dask.py

```diff
@@ -0,0 +1,302 @@
+import re
+from pathlib import Path
+from typing import Callable
+
+import dask.array as da
+import numpy as np
+import pandas as pd
+from faim_hcs.MetaSeriesUtils import (
+    _build_ch_metadata,
+    compute_z_sampling,
+    get_img_YX,
+    montage_grid_image_YX,
+)
+from faim_hcs.UIntHistogram import UIntHistogram
+from numpy._typing import ArrayLike
+from tifffile import tifffile
+
+
+
+def get_well_image_CZYX_lazy(  # noqa: C901
```

Comment:
Currently, this is a bit of a monster functions. I'll make some suggestions to simplify it / split up functionality